CRM/Contact/Form/Contact.php

```diff
@@ -996,10 +998,11 @@ public function postProcess() {
       $params['contact_id'] = $this->_contactId;
     }
 
-    //make deceased date null when is_deceased = false
-    if ($this->_contactType == 'Individual' && !empty($this->_editOptions['Demographics']) && empty($params['is_deceased'])) {
+    //make deceased date empty when is_deceased = false
+    if (($this->_contactType == 'Individual' && !empty($this->_editOptions['Demographics']) && empty($params['is_deceased']))
+      || ((($this->_contactType == 'Organization') || ($this->_contactType == 'Household')) && empty($params['is_deceased']))) {
       $params['is_deceased'] = FALSE;
-      $params['deceased_date'] = NULL;
+      $params['deceased_date'] = '';
```

Comment:
Setting deceased date to NULL did not update the existing value, but this does. Before this, clearing the deceased date was done in JS.